pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -467,24 +308,25 @@ def __init__(  # pylint: disable=too-many-arguments
             raise ImportError(
                 "Pre-compiled binaries for lightning.kokkos are not available. "
                 "To manually compile from source, follow the instructions at "
-                "https://pennylane-lightning.readthedocs.io/en/latest/installation.html."
+                "https://docs.pennylane.ai/projects/lightning/en/stable/dev/installation.html."
             )
 
-        super().__init__(wires=wires, shots=shots)
-
-        if isinstance(wires, int):
-            self._wire_map = None  # should just use wires as is
-        else:
-            self._wire_map = {w: i for i, w in enumerate(self.wires)}
+        super().__init__(
+            wires=wires,
+            c_dtype=c_dtype,
+            shots=shots,
+            batch_obs=batch_obs,
+        )
 
-        self._c_dtype = c_dtype
-        self._batch_obs = batch_obs
+        # Set the attributes to call the Lightning classes
+        self._set_Lightning_classes()
```

Comment:
I would conform to snake case here, even if Lightning is a proper name.
```suggestion
        self._set_lightning_classes()
```